packages/shorebird_code_push_api_client/test/src/shorebird_code_push_api_client_test.dart

```diff
@@ -69,7 +75,7 @@ void main() {
         expect(
           request.url,
           Uri.parse(
-            'https://shorebird-code-push-api-cypqazu4da-uc.a.run.app/api/v1/releases',
+            'https://code-push-server-kmdbqkx7rq-uc.a.run.app/api/v1/patches',
```

Comment:
Feels like we should have a single place for the base url in these expectations, no?  a helper function or something?

Reply:
yup good call 👍